python/bugstr/__init__.py

```
@@ -315,71 +451,287 @@ def _maybe_send(payload: Payload) -> None:
+        client = Client(Keys.generate())
+        client.add_relay(relay_url)
         client.connect()
-        client.send_event(gift_wrap)
+
+        # Query for the specific event by ID
+        filter = Filter().id(event_id).kind(Kind(KIND_CHUNK)).limit(1)
+        events = client.get_events_of([filter], timeout=5)
         client.disconnect()
 
+        return len(events) > 0
+    except Exception:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)